pilot/pkg/config/kube/gateway/deploymentcontroller.go

```diff
@@ -547,7 +547,9 @@ func (d *DeploymentController) setGatewayControllerVersion(gws gateway.Gateway)
 		ControllerVersionAnnotation, ControllerVersion)
 
 	log.Debugf("applying %v", patch)
-	return d.patcher(gvr.KubernetesGateway, gws.GetName(), gws.GetNamespace(), []byte(patch))
+	// Use status RBAC so we do not require full Gateway write.
+	// `status` write can modify annotations, and we already need to write status anyway so we have the permission.
```

Comment:
Looks like just a comment change; is that intended?

Reply:
The last arg changed (`status`)

Reply:
Ah, it indented to another line and I missed it; thanks